src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,119 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.DataProcessingException;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertManufacturerRequest = "INSERT INTO manufacturers(name, country) VALUES(?, ?);";
```

Comment:
For SQL it's better to use word "query" instead of "request"
```suggestion
        String insertManufacturerQuery = "INSERT INTO manufacturers(name, country) VALUES(?, ?);";
```